html/personalProfile.html

```diff
@@ -16,12 +16,12 @@
     <script src="https://www.gstatic.com/firebasejs/7.15.0/firebase-firestore.js"></script>
     <script src="../js/setupFirebase.js"></script>
     <header>
-        <h1 class="regTitle"><img src="logo1.png" class="gtlogo" alt="goodteens"></h1>
+        <h1 class="regTitle"><img src="../docs/logo1.png" class="gtlogo" alt="goodteens"></h1>
```

Comment:
normally all icons and imgs are stored in `assets` forlder.

Reply:
Will do for the next PR. It was misunderstanding - i thought no other folders were allowed to have created, so I put it there. I will know to simply create an asset folder.